src/190B20.c

```diff
@@ -1640,7 +1705,14 @@ INCLUDE_ASM(s32, "190B20", btl_appendGfx_prim_quad);
 //     gDPSetCombineMode(gMasterGfxPos++, G_CC_DECALRGBA, G_CC_DECALRGBA);
 // }
 
-INCLUDE_ASM(s32, "190B20", btl_draw_prim_quad);
+void btl_draw_prim_quad(s32 arg0, s32 arg1, s32 arg2, s32 arg3, s32 arg4, s32 arg5, s32 arg6, s32 arg7) {
+    u16 new_var;
+    u16 new_var2;
+
+    new_var = (arg4 + arg6) & 0xFFFF;
```

Comment:
Because new_var and new_var2 are of type u16, the 0xFFFF is actually implicit. You should be able to remove that from both vars